tests/phpunit/PaymentMethods/WC_Stripe_UPE_Payment_Gateway_Test.php

```
@@ -3552,4 +3553,130 @@ public function provide_test_get_excluded_payment_method_types(): array {
+			wp_deregister_script( 'wc-stripe-upe-classic' );
+			wp_deregister_style( 'wc-stripe-upe-classic' );
+
+			$gateway->payment_scripts();
+
+			$this->assertTrue( wp_script_is( 'stripe', 'enqueued' ), 'Stripe JS should be enqueued for fraud detection.' );
+			$this->assertFalse( wp_script_is( 'wc-stripe-upe-classic', 'enqueued' ), 'UPE bundle should not be enqueued when ECE is disabled.' );
+		} finally {
+			WC_Stripe_Helper::update_main_stripe_settings( $original_settings );
+			$product->delete( true );
+		}
+	}
```

PHPMD (2.15.0)

[error] The method test_payment_scripts_enqueues_only_stripe_js_on_product_page_when_ece_disabled is not named in camelCase. (undefined)

(CamelCaseMethodName)